docs/src/main/asciidoc/security-jwt-build.adoc

```diff
@@ -55,117 +58,144 @@ import org.eclipse.microprofile.jwt.JsonWebToken;
 
-One can also configure `smallrye.jwt.new-token.issuer` and `smallrye.jwt.new-token.audience` properties and skip setting the issuer and audience directly with the builder API.
+You can configure the following properties globally to avoid setting them directly in the builder:
+- `smallrye.jwt.new-token.issuer`: Specifies the default issuer.
+- `smallrye.jwt.new-token.audience`: Specifies the default audience.
 
-The next step is to decide how to secure the claims.
+After initializing and setting claims, the next step is to decide how to secure the claims.
 
 [[sign-claims]]
 == Sign the claims
 
-The claims can be signed immediately or after the `JSON Web Signature` headers have been set:
+You can sign the claims immediately or after configuring the `JSON Web Signature (JWS)` headers:
 
 [source, java]
 ----
 import io.smallrye.jwt.build.Jwt;
 ...
 
-// Sign the claims using an RSA private key loaded from the location set with a 'smallrye.jwt.sign.key.location' property.
-// No 'jws()' transition is necessary. Default algorithm is RS256.
+// Sign the claims using an RSA private key loaded from the location specified by the 'smallrye.jwt.sign.key.location' property.
+// No 'jws()' transition is required. The default algorithm is RS256.
 String jwt1 = Jwt.claims("/tokenClaims.json").sign();
 
-// Set the headers and sign the claims with an RSA private key loaded in the code (the implementation of this method is omitted).
-// Note a 'jws()' transition to a 'JwtSignatureBuilder', Default algorithm is RS256.
-String jwt2 = Jwt.claims("/tokenClaims.json").jws().keyId("kid1").header("custom-header", "custom-value").sign(getPrivateKey());
+// Set the headers and sign the claims by using an RSA private key loaded in the code (the implementation of this method is omitted).
+// Includes a 'jws()' transition to a 'JwtSignatureBuilder'. The default algorithm is RS256.
+
+String jwt2 = Jwt.claims("/tokenClaims.json")
+                 .jws()
+                 .keyId("kid1")
+                 .header("custom-header", "custom-value")
+                 .sign(getPrivateKey());
 ----
 
-Note the `alg` (algorithm) header is set to `RS256` by default. Signing key identifier (`kid` header) does not have to be set if a single JSON Web Key (JWK) containing a `kid` property is used.
+Default behaviors:
+
+- The `alg` (algorithm) header is set to `RS256` by default.
+- You do not have to set a signing key identifier (`kid` header) if a single JSON Web Key (JWK) containing a `kid` property is used.
 
-RSA and Elliptic Curve (EC) private keys as well as symmetric secret keys can be used to sign the claims.
-`ES256` and `HS256` are the default algorithms for EC private and symmetric key algorithms respectively.
+Supported keys and algorithms:
 
-You can customize the signature algorithm, for example:
+- To sign the claims, you can use RSA private keys, Elliptic Curve (EC) private keys, and symmetric secret keys.
+- `RS256` is the default algorithm for RSA private keys.
```

Comment:
```suggestion
- `RS256` is the default algorithm for RSA private key signature algorithms.
```

Reply:
Thank you. Updated per our Slack conversation. Please let me know if you have any further suggestions or feedback.